DOC-2270: Add improved documentation for TINY-10474 in TinyMCE 7.0 release notes (#3147)
tinymce/tinymce-docs: modules/ROOT/pages/7.0-release-notes.adoc

[NOTE]
Interactive elements with visible labels do not display custom tooltips, and the browser native tooltips have been removed, as the visible label provides sufficient information.

[TIP]

> The `max-width` of the tooltips is set to `15em`, to overwrite the default `max-width` of the tooltips, use the `@tooltip-max-width` oxide variable.

Reviewer: ```suggestion
The `max-width` of the tooltips is set to `15em`, to overwrite the default `max-width` of the tooltips, update the `@tooltip-max-width` variable in your custom skin.
```

As of {productname} 7.0, the tooltip property is now optional for `DialogFooterToggleButton`. This flexibility allows developers to omit the tooltip for buttons where it's not required, providing a cleaner and more focused user experience.

=== Custom tooltips for `forecolor`, `backcolor` buttons and bespoke select toolbar buttons including `align`, `fontfamily`, `fontsize`, `blocks`, and `styles`
// #TINY-10474

As of {productname} 6.8.2, tooltips for several text formatting buttons in {productname} displayed information based on the following behavior:

* `forecolor` and `backcolor`: The last used color was shown in the tooltip.
* `align, `fontfamily`, `fontsize`, `blocks`, and `styles`: The currently selected value was displayed.

> {productname} 7.0, introduces custom tooltips for these buttons. These tooltips dynamically adapt based on the button's current state and the currently selected value. They are now visible on both mouse hover and keyboard focus, improving accessibility.

Reviewer: The new custom tooltips in 7 apply to all buttons, including these, so is it necessary to include this as a separate entry?

Author: I've removed them yeah since they're covered in TINY-10453, no need of duplication.
